kernel/build-kernel.sh

```diff
@@ -201,10 +201,8 @@ get_kernel_frag_path() {
 	results=$( ${cmdpath} -r -n ${all_configs} )
 	# Only consider results highlighting "not in final"
 	results=$(grep "${not_in_string}" <<< "$results")
-	# Do not care about options that are in whitelist if using experimental kernel
-	if [[ ${experimental_kernel} == "true" ]]; then
-		results=$(grep -v -f ${default_config_whitelist} <<< "$results")
-	fi
+	# Do not care about options that are in whitelist
+	results=$(grep -v -f ${default_config_whitelist} <<< "$results")
```

Comment:
could you move `fragments/common/experimental/virtio-fs.conf` into `fragments/common/`?

Reply:
@devimc Are the virtio-fs bits merged in stable?
cc @egernst

Reply:
nop, I didn't see this, ignore my comment

https://github.com/kata-containers/runtime/blob/1fa10fea78e88a520652463189248d91145518f2/versions.yaml#L160-L163